requirements-dev.txt

```diff
@@ -312,7 +312,7 @@ ply==3.11
     #   jsonpath-ng
     #   jsonpath-rw
     #   pandoc
-pre-commit==3.8.0
+pre-commit==4.0.0
```

Comment:
https://github.com/pre-commit/pre-commit/releases

Will have to update our pre-commit hook at some point to deal with the deprecations I think

Reply:
Okay, without looking at the release, is that something that should / could be done in this PR?

Reply:
changes would mostly need to happen in the hook repo itself, but I checked the deprecations now and there is nothing to do after all :)